assets/dev/js/editor/document/commands/base/command-history.js

```diff
@@ -42,7 +42,8 @@ export default class CommandHistory extends CommandBase {
 	 * @returns {boolean}
 	 */
 	isHistoryActive() {
-		return elementor.documents.getCurrent().history.getActive();
+		return elementor.documents.getCurrent().history.getActive() &&
+			false !== $e.commands.getCurrentFirstTraceArgs().options?.useHistory;
```

Comment:
Would be great if you can add test in `CommandHistory.spec` that covers `useHistory`